lib/round.rb

```diff
@@ -0,0 +1,78 @@
+class Round
+  attr_reader :deck, :turns, :current_card, :current_turn, :number_correct,:cat_choice_qs, :correct_qs, :num_right
+
+  def initialize(deck)
+    @deck = deck #deck of cards for this round
+    @turns = [] #initialized with empty array
+    @current_turn = 0 #to iterate through deck depending on turn
+    @number_correct = 0 #inititalized with 0
+    @correct_qs = [] #placeholder array for correct answers
+
+  end
+
+
+  def current_card #moved to individual method to make sure it updated the iteration
+    @current_card = @deck.cards[@current_turn]#initializes current card with first card of deck
+    return @current_card
+  end
```

Comment:
This works! Something else that might be a little more straightforward and easier to read would be something like this:
```ruby
def current_card
    deck.cards.first
  end
```

And then physically rotate/shuffle the deck after the user takes a turn. That way, you don't also have to keep track of the `@current_turn`.

```ruby
  def rotate_deck
    deck.cards.rotate!
  end
```